src/lib/components/EventsPanel.svelte

```diff
@@ -20,24 +21,25 @@ import Card from './Card.svelte';
 		<p class="text-center text-gray-600 mb-4">{description}</p>
 	{/if}
 	<div class="grid grid-cols-1 gap-2">
-		{#each events as event}
-			<div class="flex items-center p-2 border rounded-lg">
-				<CalendlyPopupButton bookingLink={event.bookingLink}>Book</CalendlyPopupButton>
-				<!-- <div class="w-16 h-16 bg-gray-100 rounded-lg overflow-hidden">
-					<img src={event.imageSrc} alt={event.title} class="object-cover w-full h-full" />
-				</div> -->
-				<div class="ml-4 flex-grow">
-					<h3 class="font-semibold">{event.title}</h3>
-					<p class="text-sm text-gray-600 truncate">{event.description}</p>
+		{#if events.length > 0}
+			{#each events as event}
+				<div class="flex items-center p-2 border rounded-lg">
+					<CalendlyPopupButton bookingLink={event.bookingLink}>Book</CalendlyPopupButton>
+					<div class="ml-4 flex-grow">
+						<h3 class="font-semibold">{event.title}</h3>
+						<p class="text-sm text-gray-600 truncate">{event.description}</p>
+					</div>
+					<!-- <div class="ml-4 min-w-fit">
```

Comment:
Removed the date section for now since we aren't sure how we will handle this. We'll see if it's something they want -- for now I think it looks fine with just the button and it can now display a longer description!